src/components/o-commercial-hero-cover/index.scss

```diff
@@ -0,0 +1,115 @@
+@import "../../styles/typo/typo-mixins";
+@import "../../styles/settings/colors";
+@import "../../styles/grid/grid-mixins";
+@import "../../styles/mixins/respond";
+@import "../../styles/typo/typo-mixins";
+@import "../../styles/mixins/word-break";
+@import "../m-button/mixins/button";
+
+.o-commercial-hero-cover {
+  display: block;
+
+  .o-commercial-hero-cover__header {
+    position: relative;
+    min-height: 256px;
```

Comment:
ok am sorry i forgot to say it, but we use variables to be defined on top of the scss file instead of just values inside the scss for maintainability reasons. Could you fix that please and sorry.